go.mod

```diff
@@ -1,30 +1,30 @@
 module github.com/openshift/client-go
 
-go 1.20
+go 1.21.3
```

Comment:
Same comment as in the api, let's stick with 1.20 for now.

Reply:
Addressed